cli/tests/integration/lsp_tests.rs

```diff
@@ -3547,7 +3609,7 @@ fn lsp_performance() {
     .unwrap();
   assert!(maybe_err.is_none());
   if let Some(res) = maybe_res {
-    assert!(res.averages.len() >= 6);
+    assert!(res.averages.len() >= 5);
```

Comment:
5 now because there's no performance trace for the overall diagnostics since they all happen independently now. I wonder if this should be `assert_eq!(res.averages.len(), 5);`?

Reply:
It used to be somewhat fragile with `== 6` IIRC and so it was changed to `>= 6`, but I guess we can try now.

Reply:
Ok, let's try it and see how it goes.